MAUI/Polar-Charts/Exporting.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Exporting in .NET MAUI Chart control | Syncfusion
+description: Learn here how to export the chart view as an image and stream in the Syncfusion .NET MAUI Chart (SfPolarChart) control.
+platform: maui
+control: SfPolarChart
+documentation: ug
+---
+
+# Exporting in .NET MAUI Chart
+
+## Export as an image
+
+You can export the chart view as an image in the desired file format using the [SaveAsImage]() method of [SfPolarChart](). The supported image formats are **JPEG and PNG**. By default, if you don't mention any image format with the filename, the chart view will be exported as an image in the PNG format.
+
+N> The chart view can be exported as an image only when the chart view is added to the visual tree.
+
+The following code snippet demonstrates the usage of this method:
+
+{% tabs %}
+
+{% highlight c# %}
+
+ SfPolarChart chart = new SfPolarChart();
+ ...
+ chart.SaveAsImage("ChartSample.jpeg");
+
+this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+T> We can change the image formats in the above code by changing their extensions to .jpg or .png.
+
+The exported image will be saved in the different location across the platforms.
+
+**Windows Phone, Android and MAC** – The image will be saved inside the 'Pictures' directory of the file system.
```

Comment:
**Windows Phone, Android, and MAC** – The image will be saved inside the 'Pictures' directory of the file system.